module/VuFind/src/VuFind/Auth/MultiILS.php

```diff
@@ -80,6 +81,18 @@ public function authenticate($request)
         }
 
         // Did the patron successfully log in?
+        if ('email' === $loginMethod) {
+            if (null === $this->emailAuthenticator) {
```

Comment:
This new chunk of code appears to be exactly duplicated from the parent class. Do you think it's worth creating a protected handleEmailLogin() support method to minimize duplication (and perhaps ease future changes, if we decide on a better mechanism than exception throwing for notification of in-progress logins).